### ISSUE TITLE
User remove list

### ISSUE THREAD

**Issue by rdutta** [unexpectedpanda/retool#102]
Wondering how to interpret this output,

```
This file shows which titles have been removed from the output dat
due to the user setting an option.

TITLE TYPES REMOVED
===================
* CATEGORIES


# CATEGORIES
============
- GameBooster 64 (Europe) (v1.1) (Unl)
- GameBooster 64 (USA) (v1.1) (Unl)
- GameShark Pro (USA) (v2.0) (Unl)
- GameShark Pro (USA) (v2.0) (Unl)
- GameShark Pro (USA) (v3.3) (Unl)
- iQue Club (China) (v3) (iQue)
```

I have no custom filters and nothing excluded as far as I can tell. I'm wondering what this output means and where it got those Gameshark and iQue "Categories" from. Thanks.

**Retool edition**
retool-gui.exe

**Retool version**
v0.96

**Are you using custom global or system filters? If so, list them below**
None

**What other settings are you using?**
Suggested region order for English Speakers

**Comment by unexpectedpanda** [comment]
Looks like an unintended side effect of the new feature that manually assigns categories in clone lists.

As far as I can tell the titles aren't removed, just erroneously added to the log. Otherwise things are operating properly.

I'll get this fixed in the next release.

**Comment by unexpectedpanda** [comment]
Fixed in 0.97 :)
